docs/getting-started/istio/index.md

```diff
@@ -33,15 +33,19 @@ spec:
           virtualService:
             # Reference to a VirtualService which the controller updates with canary weights
             name: rollouts-demo-vsvc
+            # optional if there is a single route in VirtualService, required otherwise
             routes:
-            - primary # optional if there is a single route in VirtualService, required otherwise
+            - http-primary
+            - https-3000
 ...
 ```
 
 The VirtualService and route referenced in `trafficRouting.istio.virtualService` is required
-to have a HTTP route which splits between the stable and canary Services, referenced in the rollout.
-In this guide, those Services are named: `rollouts-demo-stable` and `rollouts-demo-canary` 
-respectively. The weight values for these services used should be initially set to 100% stable, 
+to have either an HTTP or a TLS route spec that splits between the stable and canary Services,
+referenced in the rollout. If the route is HTTP/TLS route it of type then it needs to be
```

Comment:
Shouldn't this sentence read: "If the route is HTTPS/TLS route"

(missing S in HTTPS)

Reply:
That's correct! Thanks for pointing it out.